### PR TITLE
Added ValuesController from WebApi base app as an ApiController exmaple

### PR DESCRIPTION
**Pull request by thyeggman**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi **@jwallra**, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: backend/dotnet/Quickstart/ZUMOAPPNAMEService/Controllers/ValuesController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web.Http;
+using Microsoft.Azure.Mobile.Server.Config;
+
+namespace ZUMOAPPNAMEService.Controllers
+{
```
Comment (brettsam) [inline backend/dotnet/Quickstart/ZUMOAPPNAMEService/Controllers/ValuesController.cs:8]: We may want to add some comment here describing with the [MobileAppController] attribute does -- something like "Use the MobileAppControllerAttribute to identify ApiControllers that you want to use from your mobile clients".

Comment (mattchenderson) [inline backend/dotnet/Quickstart/ZUMOAPPNAMEService/Controllers/ValuesController.cs:8]: Agreed on the comment.

Comment (thyeggman) [inline backend/dotnet/Quickstart/ZUMOAPPNAMEService/Controllers/ValuesController.cs:8]: Fixed

**Pull request merged**